zerver/decorator.py

```diff
@@ -388,7 +387,7 @@ def redirect_to_login(next: str, login_url: Optional[str]=None,
 
     return HttpResponseRedirect(urllib.parse.urlunparse(login_url_parts))
 
-# From Django 1.8
+# From Django 2.2
```

Comment:
Since the code is not modified from upstream. I think it's better to use the version from Django itself or are we planning to modify it?